zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -272,9 +290,9 @@ public static IvParameterSpec getKeyringInitVector() {
      *
      * @param key The name of the configuration value.
      * @return The system property corresponding to the given key (e.g.
-     *         -Dkey=value). If that is blank, the environment variable
```

Comment:
I don't think these changes are needed, I think that the Javadoc was correct before.